src/transformers/models/falcon/modeling_falcon.py

```diff
@@ -469,39 +473,50 @@ def forward(
                 return output_tensor, present
 
         else:
-            matmul_result = query_layer @ key_layer.transpose(-1, -2)
+            if hasattr(F, "scaled_dot_product_attention") and not output_attentions and head_mask is None:
+                context_layer = torch.nn.functional.scaled_dot_product_attention(
+                    query_layer,
+                    key_layer,
+                    value_layer,
+                    attn_mask=attention_mask,
+                    dropout_p=self.attention_dropout.p if self.training else 0.0,
+                    is_causal=self.is_causal and attention_mask is None and query_length > 1,
+                )
+                context_layer = context_layer.transpose(1, 2)
+                context_layer = context_layer.reshape(batch_size, query_length, self.num_heads * self.head_dim)
+
+                output_tensor = self.dense(context_layer)
```

Comment:
would be nice to standardize the naming here and there

Reply:
done thank you